core/embed/sys/mpu/stm32u5/mpu.c

```diff
@@ -135,8 +135,10 @@ _Static_assert(NORCOW_SECTOR_SIZE == STORAGE_2_MAXSIZE, "norcow misconfigured");
 // Extended peripheral block to cover FMC1 that's used for display
 // 512M of periherals + 16M for FMC1 area that follows
 #define PERIPH_SIZE (SIZE_512M + SIZE_16M)
+#define PERIPH_SIZE_EXT (SIZE_256M + SIZE_16M)
```

Comment:
this is a bit confusing, in EXT variant we actually cover less area than normal variant (we still cover non-secure `FLASH` periph but not all non-secure periphs as with `U5G` and we don't cover `FMC1`). I think we should either set  `PERIPH_SIZE_EXT` to `SIZE_512M + SIZE_16M` for `U585`, or to `SIZE_256M + SIZE_16M` for `U5G`.